### PR TITLE
fix pagination range issue

### PR DESCRIPTION
**Pull request by Mustufa-Coder**
- pagination range value provided with pagination max size is not working

### CODE REVIEW

File: src/app/demo/pagination-range/pagination-range.component.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export class PaginationRangeComponent implements OnInit {
 
   ngOnInit(): void {
     this.configuration = { ...DefaultConfig };
-    this.configuration.paginationMaxSize = 7;
+    this.configuration.rows = 7;
```
Comment (ssuperczynski) [inline src/app/demo/pagination-range/pagination-range.component.ts:45]: I just checked this:
```ts
this.configuration.paginationMaxSize = 7;
this.configuration.rows = 3;
```
And this works fine. What is the issue here?

Screenshot 2021-11-08 at 10 03 34 AM

Comment (Mustufa-Coder) [reply src/app/demo/pagination-range/pagination-range.component.ts:45]: image

Here pagination size is 7 but still showing 10 records, 
** this.configuration.rows = 7;** - this line is missing in code

Comment (ssuperczynski) [reply src/app/demo/pagination-range/pagination-range.component.ts:45]: Thanks, 
`paginationMaxSize` is a property where you define the maximum number of pages in the pagination. So when you set it to 3 it is like an image below. `rows` is a property to set an aomut of the rows in the table.

Screenshot 2021-11-08 at 2 24 11 PM
s

Comment (Mustufa-Coder) [reply src/app/demo/pagination-range/pagination-range.component.ts:45]: Okay, thanks

**Pull request closed without merging**